src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -141,7 +149,21 @@ class GcsPlacementGroupManager : public rpc::PlacementGroupInfoHandler {
   void OnPlacementGroupCreationSuccess(
       const std::shared_ptr<GcsPlacementGroup> &placement_group);
 
+  /// Handle a node death. This will reschedule all bundles associated with the
+  /// specified node id.
+  ///
+  /// \param node_id The specified node id.
+  void OnNodeDead(const ClientID &node_id);
+
  private:
+  /// Scheduling priority of placement group: RESCHEDULING > PENDING.
```

Comment:
So, our priority will be 
RESCHEDULING
PENDING
WAITING

Is it correct?

Reply:
yeah, it is correct.